### PR TITLE
Add checks from blueprint test_component,

### PR DESCRIPTION
**Pull request by akokoshn**
allow fill assignment tables for exactly one circuit

### CODE REVIEW

**Comment by akokoshn** [comment]
waiting for: https://github.com/NilFoundation/evm-assigner/pull/22

File: bin/assigner/include/checks.hpp [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    }
+    bool is_connected = nil::blueprint::check_connectedness(
+        assignment, bp, instance_input, component_result, start_row,
+        rows_after_batching - start_row, connectedness_check);
+    if (!is_connected) {
+        std::cout << "ERROR: connectness violation\n";
+        return false;
+    }
+    if (connectedness_check.t == nil::blueprint::connectedness_check_type::type::NONE) {
+        std::cout << "WARNING: Connectedness check is disabled.\n";
```
Comment (makxenov) [inline bin/assigner/include/checks.hpp:53]: The same

File: bin/assigner/include/checks.hpp [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    }
+    return true;
+}
+
+template<typename BlueprintFieldType>
+bool is_satisfied(
```
Comment (makxenov) [inline bin/assigner/include/checks.hpp:59]: Also unused

File: bin/assigner/src/main.cpp [hunk 3]
```diff
@@ -74,6 +77,11 @@ int curve_dependent_main(const std::string& input_block_file_name,
         std::cerr << "Assigner run failed: " << run_err.value() << std::endl;
         return 1;
     }
+
+    /*if (!check_assignment_tables()) {
```
Comment (makxenov) [inline bin/assigner/src/main.cpp:81]: Commented code

File: bin/assigner/include/checks.hpp [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+#ifndef ZKEMV_FRAMEWORK_LIBS_CHECKS_INCLUDE_CHECKS_HPP_
+#define ZKEMV_FRAMEWORK_LIBS_CHECKS_INCLUDE_CHECKS_HPP_
+
+#include <nil/blueprint/blueprint/plonk/assignment.hpp>
+#include <nil/blueprint/blueprint/plonk/circuit.hpp>
+#include <nil/blueprint/utils/connectedness_check.hpp>
+#include <nil/blueprint/utils/satisfiability_check.hpp>
+
+template<typename BlueprintFieldType>
+bool check_connectness(
```
[the remark this answers is not in the document]
Comment (akokoshn) [reply bin/assigner/include/checks.hpp:16]: done

[the remark this answers is not in the document]
Comment (akokoshn) [reply bin/assigner/include/checks.hpp:16]: Yes, i just allow to use these check if need

File: bin/assigner/include/checks.hpp [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+    const std::vector<
+        nil::crypto3::zk::snark::plonk_variable<typename BlueprintFieldType::value_type>>&
+        instance_input,
+    const std::vector<
+        nil::crypto3::zk::snark::plonk_variable<typename BlueprintFieldType::value_type>>&
+        component_result,
+    size_t start_row, size_t component_rows_amount,
+    nil::blueprint::connectedness_check_type connectedness_check) {
+    if (assignment.rows_amount() - start_row != component_rows_amount) {
+        std::cout << "ERROR: wrong rows ampunt: expected " << component_rows_amount << ", real "
```
[the remark this answers is not in the document]
Comment (akokoshn) [reply bin/assigner/include/checks.hpp:30]: done

File: bin/assigner/include/checks.hpp (hunk 1 above)
Comment (akokoshn) [reply bin/assigner/include/checks.hpp:53]: done

File: bin/emulator/tmp/main.cpp [hunk 6]
```diff
@@ -0,0 +1,5 @@
+#include "wrapper.hpp"
```
[the remark this answers is not in the document]
Comment (akokoshn) [reply bin/emulator/tmp/main.cpp:1]: good catch

File: bin/assigner/src/main.cpp (hunk 3 above)
Comment (akokoshn) [reply bin/assigner/src/main.cpp:81]: That is an example where should be called check assignment tables functions

**Review by makxenov — APPROVED** [review]

**Pull request merged**